File: assignments/Regular/cartax.py
print("Program calculates road tax")

car_price = int(input("How much is the car worth? : "))

if(car_price < 1_000_000):
	tax = car_price * (10 / 100)

if(car_price > 1_000_000 and car_price < 3_000_000):
	tax = car_price * (15 / 100)

if(car_price > 3_000_000 and car_price < 5_000_000):
	tax = car_price * (20 / 100)

if(car_price > 5_000_000):
	tax = car_price * (25 / 100)

print(f"Your tax on your car that cost {car_price} is {tax}")

File: Assignment JAva/Regular/CarTax.java
import java.util.Scanner;

public class CarTax{
	public static void main(String...args){
	Scanner input = new Scanner(System.in);
		
		System.out.print("This program calculates the tax on cars");

		//int flag = -1;
		double tax = 0;

		//while(flag != -1){
			System.out.print("How much is the car worth? : ");	
			double carPrice = input.nextInt();

			if(carPrice < 1_000_000){
				 tax = carPrice * (10 / 100);
			}

			if(carPrice > 1_000_000 && carPrice < 3_000_000){
				 tax = carPrice * (15/ 100);
			}

			if(carPrice > 3_000_000 && carPrice < 5_000_000){
				 tax = carPrice * (20 / 100);
			}

			if(carPrice > 5_000_000){
				 tax = carPrice * (25 / 100);
			}


			System.out.print("Your tax on your car that cost "+tax);
			
			//System.out.print("Press 1 to continue or -1 to exit: ");	
			//flag = input.nextInt();


		//}
	
		
	}
}